build.sh

```diff
@@ -273,17 +273,22 @@ while [[ $# -gt 0 ]]; do
         _TARGET_OS="${_TARGET_OS:9}"
         if [[ $_TARGET_OS =~ "android" ]]; then
             OLD_PATH=$PATH
```

Comment:
Better move this line too?

Reply:
I will keep this, because this is **target** option. consider android is not official supported, it could be a **temporary** solution.

I would wish this option has ability like android-configure:
1. check external toolchain (set by android-configure) and apply
2. build standalone toolchain if toolchain undefined and set PATH

Reply:
@miaulightouch what I mean is... You moved everything but `OLD_PATH=$PATH` into an `IF` section. Only use I can see for `OLD_PATH` is inside that `IF` section.

So, could you please move that line too?

Reply:
Or.. if you are concerned with the TARGET_OS thing.. you may move it up too.